app/commands/v2/unpublish.rb

```diff
@@ -44,7 +44,7 @@ def call
           raise_command_error(422, message, fields: {})
         end
 
-        Success.new(content_id: content_id)
+        [Success.new(content_id: content_id), callback]
```

Comment:
Is the receiving code going to be OK with `callback` being `nil` for redirects and gones?